derive-input/src/internal/table.rs

```diff
@@ -50,11 +63,32 @@ pub(in crate::internal) fn try_parse(
     })
 }
 
+// Parse plural_name from DSL arguments
+fn parse_plural_name_from_args(args: &proc_macro2::TokenStream) -> syn::Result<Option<Ident>> {
+    use crate::internal::dsl::plural_name;
+    
+    let mut plural_name_value: Option<Ident> = None;
+
+    parser(|meta| {
+        match_meta!(match meta {
+            plural_name => {
+                check_duplicate(&plural_name_value, &meta)?;
+                let value = meta.value()?;
+                plural_name_value = Some(value.parse()?);
+            }
+        });
+        Ok(())
+    })
+    .parse2(args.clone())?;
+
+    Ok(plural_name_value)
+}
+
```

Comment:
remove this function, it's a duplication of the parse_plural_name_from_args function in the derive-input/src/internal/table.rs file.